nikola/data/themes/base/templates/index.tmpl

```diff
@@ -11,6 +11,14 @@
 </%block>
 
 <%block name="content">
+%if subcategories:
```

Comment:
That's because `tag_index.tmpl` lets `index.tmpl` do all the work. Usually `subcategories` is not set, so nothing will happen.
It would be better to move this into `tagindex.tmpl`, but I was too lazy for that during the first try.